src/crsAdapter/TomaAdapter.js

```diff
@@ -696,10 +701,11 @@ class TomaAdapter {
 
         let travellerLineNumber = void 0;
 
-        service.travellers.forEach((traveller) => {
+        service.travellers.forEach((ServiceTraveller) => {
```

Comment:
do not use upper case as first letter for a variable

Reply:
done.